website/content/labs/lab2.md

```diff
@@ -76,17 +61,10 @@ In this task, you must do the following:
 
     In the `test_point.c` file, update the intersection tests to make sure they use the new version of the file (you don’t need to know exactly how these tests work, and you can keep these tests inside the `test_point.c`; in a future lab, we will see a cleaner way of updating the tests). Once you have fixed this, you should be able to run `make tests` followed by `tests/test-libgeometry`.
 
-## Project Team Exercise: Design Warm-up
-
-You can find the specification of the team exercise [here]({{< relref "../projects/2019/design-1.md" >}})
-
-Please note that this exercise is not part of your Lab #2 grade. Instead, it will be part of your Project grade for the class (see the [syllabus]({{< relref "syllabus.md" >}}) for more details) You should nonetheless use the lab time to get to know your team, and to start discussing the team exercise.
-
 
 ## Submitting your lab
 
-Before submitting, make sure you've added, committed and pushed all your work in the `lab2` directory (remember you can run `git status` to check this). Make sure you've set up the `chisubmit` tool as described in [How to submit your labs]({{< relref "submit.md" >}}), and then run the following:
+Before submitting, make sure you've pushed all your code to GitHub. You will submit your code through GitHub but, unlike the previous lab, we need you to submit the code that is in your repository. When submitting through Gradescope, you will be given the option of manually uploading files, or of uploading a GitHub repository (we recommend the latter, as this ensures you are uploading exactly the files that are in your repository). Please note that you can submit as many times as you want before the deadline.
```

Comment:
Might be worth changing to "make sure you've added, committed, and pushed..."

Also, add: "select your 2020-lab2-GIT_USERNAME repository, with "master" as the branch

Reply:
Pushed a fix for this.